### DISCUSSION TITLE
Welcome to ID3TagEditor Discussions!

### DISCUSSION THREAD

**Discussion by chicio** [chicio/ID3TagEditor#77] [General]
## 👋 Welcome to ID3TagEditor Discussions 🎧🎵!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask❓questions ❓you’re wondering about how ID3TagEditor works.
  * Share 🧠 ideas 🧠.
  * Be kind to all the other members. This is a community that we want to build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community and what you like about ID3TagEditor ♥️.

**Comment by lesterrry** [comment]
Hello. I'm working with MP3 CDs and this library helps a lot. However, the `read` function, which returns the tag, gathers all the info across the file, which takes some time and makes disk spin very noisy. I wonder whether it's possible to only read particular Frames from a file? For example, title, album and artist.
Thank you.

**Reply by chicio** [reply]
Hi @Lesterrry ,

sorry for the late reply. No, at the moment this api doesn't exist in ID3TagEditor. I have it planned but without a certain release date. If you want you can try to make a contribution and help me with the development by opening a MR.

Thanks,
Fabrizio.

**Comment by KentonTsai** [comment]
Hello. I'm working with MP3 file. After i use Chinese String to **write** Genre data, it will become Garbled String.Please help to check it.Thanks.

**Reply by chicio** [reply]
Hi @KentonTsai ,

can you open an issue with the details of your problem (e.g a mp3 file without copyright with the id3 tag that is not parsed).

Thanks,
Fabrizio.
